### PR TITLE
Add preference to hide the enclosure view

### PR DESCRIPTION
**Pull request by Eitot**
This was suggested here: https://forums.cocoaforge.com/viewtopic.php?f=18&t=27230.

### CODE REVIEW

**Review by barijaona — APPROVED** [review]

File: Vienna/Sources/Main window/ArticleListView.m [hunk 1]
```diff
@@ -1655,17 +1670,39 @@ -(NSArray *)markedArticleRange
...
+        [super observeValueForKeyPath:keyPath
+                             ofObject:object
+                               change:change
+                              context:context];
+        return;
+    }
+
+    if ([keyPath isEqualToString:MAPref_ShowEnclosureBar]) {
+        NSNumber *showEnclosureBar = change[NSKeyValueChangeNewKey];
+        if (showEnclosureBar.boolValue) {
```
Comment (barijaona) [inline Vienna/Sources/Main window/ArticleListView.m:1698]: Cf. -refreshArticlePane : showing the enclosure view should be done if : 
- just one article is displayed
- the article has an enclosure

**Review by barijaona — CHANGES_REQUESTED** [review]

File: Vienna/Sources/Main window/ArticleListView.m [hunk 2]
```diff
@@ -1655,17 +1670,39 @@ -(NSArray *)markedArticleRange
...
+            [self showEnclosureView];
```
Comment (barijaona) [inline Vienna/Sources/Main window/ArticleListView.m:1699]:
```suggestion
            [self refreshArticlePane];
```

**Review by barijaona — CHANGES_REQUESTED** [review]
For consistency, showing the enclosure view should only be done if :
- just one article is displayed
- the article has an enclosure

**Review by barijaona — APPROVED** [review]

**Comment by barijaona** [comment]
> I did not even know that you can view multiple articles at once.

Yeah, it's one of the features I like most in Vienna, for quickly scrolling through folders containing photos thumbnails…

Neither NetNewsWire or Reeder have this.

**Comment by barijaona** [comment]
@Eitot : the new menu item "Hide Status Bar" has not yet been exported to Crowding for localization…
Is there any reason ?

**Comment by barijaona** [comment]
I understand. But it will leave final versions of 3.8.x looking half finished 😀. In the future, we should coordinate a little more regarding merging of features branches.

**Comment by josh64x2** [comment]
@barijaona co-ordination is definitely welcome, do you have any method in mind?

**Comment by barijaona** [comment]
@Eitot thanks !
@josh64x2 how about a "requests translation" label for PRs ?

**Pull request merged**